src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/UserManagementServiceTests.cs

```diff
@@ -25,535 +23,780 @@
+                userRoleService.Verify(mock => mock.GetPublicationRolesForUser(userId), Times.Once);
+                userRoleService.Verify(mock => mock.GetReleaseRoles(userId), Times.Once);
+            }
+        }
+
+        [Fact]
+        public async Task NoUser_ReturnsNotFound()
         {
+            var userRoleService = new Mock<IUserRoleService>(Strict);
+
+            await using var userAndRolesDbContext = InMemoryUserAndRolesDbContext();
+
             var service = SetupUserManagementService(
                 usersAndRolesDbContext: userAndRolesDbContext,
                 userRoleService: userRoleService.Object
             );
 
-            var result = await service.GetUser(userId);
+            var result = await service.GetUser(Guid.NewGuid());
 
             VerifyAllMocks(userRoleService);
 
-            Assert.True(result.IsRight);
-            var userViewModel = result.Right;
-
-            Assert.Equal(userId, userViewModel.Id);
-            Assert.Equal("test@test.com", userViewModel.Email);
-            Assert.Equal("Test Analyst", userViewModel.Name);
-            // Currently we only allow a user to have a maximum of one global role
-            Assert.Equal(globalRoles[0].Id, userViewModel.Role);
-            Assert.Equal(publicationRoles, userViewModel.UserPublicationRoles);
-            Assert.Equal(releaseRoles, userViewModel.UserReleaseRoles);
-
-            userRoleService.Verify(mock => mock.GetGlobalRoles(user.Id), Times.Once);
-            userRoleService.Verify(mock => mock.GetPublicationRolesForUser(userId), Times.Once);
-            userRoleService.Verify(mock => mock.GetReleaseRoles(userId), Times.Once);
+            result.AssertNotFound();
         }
     }
 
-    [Fact]
-    public async Task GetUser_NoUser()
+    public class ListPendingInvitesTests : UserManagementServiceTests
     {
-        var userRoleService = new Mock<IUserRoleService>(Strict);
+        [Fact]
+        public async Task Success()
+        {
+            var identityRole = new IdentityRole { Name = GlobalRoles.RoleNames.Analyst };
+
+            var pendingUserInvites = _dataFixture.DefaultUserWithPendingInvite().WithRole(identityRole).GenerateList(3);
+
+            var activeUser = _dataFixture.DefaultUser().Generate();
+
+            var softDeletedUser = _dataFixture.DefaultSoftDeletedUser().Generate();
+
+            var expiredUser = _dataFixture.DefaultUserWithExpiredInvite().Generate();
+
+            var publication = _dataFixture
+                .DefaultPublication()
+                .WithReleases([_dataFixture.DefaultRelease(publishedVersions: 1)])
+                .Generate();
+
+            var userReleaseInvites = _dataFixture
+                .DefaultUserReleaseInvite()
+                .WithReleaseVersion(publication.Releases[0].Versions[0])
+                .ForIndex(0, s => s.SetEmail(pendingUserInvites[0].Email))
+                .ForIndex(1, s => s.SetEmail(pendingUserInvites[0].Email))
```

Comment:
I'm not sure that I follow why each email has a duplicate invite? Is that a possibility?